parm/config/gfs/config.ufs

```diff
@@ -308,9 +339,13 @@ if [[ "${skip_mom6}" == "false" ]]; then
   export DT_DYNAM_MOM6 DT_THERM_MOM6
   export FRUNOFF
   export CHLCLIM
+  export TOPOEDITS
   export MOM6_RIVER_RUNOFF
   export MOM6_RESTART_SETTING
   export eps_imesh
+  export MOM6_DIAG_COORD_DEF_Z_FILE
+  export MOM6_DIAG_MISVAL
+  export MOM6_ALLOW_LANDMASK_CHANGES
```

Comment:
These need to be added to the GEFS version. This is what is causing the CI GEFS test to fail:

```
/scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1991/global-workflow/ush/parsing_namelists_MOM6.sh: line 66: MOM6_ALLOW_LANDMASK_
CHANGES: unbound variable
```

Reply:
done